libs/explorers/ui/src/lib/components/search-input/search-input.component.ts

```diff
@@ -104,7 +105,7 @@ export class SearchInputComponent implements AfterViewInit {
           const keyEvent = event as KeyboardEvent;
           return !['ArrowDown', 'ArrowUp', 'Enter', 'Escape'].includes(keyEvent.key);
         }),
-        debounceTime(500),
+        debounceTime(DEBOUNCE_TIME_MS),
```

Comment:
Using a constant as we had different debounce times configured in different parts of the code.